From c412abbabcdac78bcb461e7de53ea6c8d21eb30c Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sat, 15 Apr 2023 10:53:09 -0700
Subject: [PATCH] Fix libretro print screen button

---
 dosbox_pure_libretro.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dosbox_pure_libretro.cpp b/dosbox_pure_libretro.cpp
index 6a03631..9b21b9a 100644
--- a/dosbox_pure_libretro.cpp
+++ b/dosbox_pure_libretro.cpp
@@ -3964,7 +3964,7 @@ void retro_init(void) //#3
 		{RETROK_EQUALS, KBD_equals}, {RETROK_BACKSLASH, KBD_backslash}, {RETROK_LEFTBRACKET, KBD_leftbracket},
 		{RETROK_RIGHTBRACKET, KBD_rightbracket}, {RETROK_SEMICOLON, KBD_semicolon}, {RETROK_QUOTE, KBD_quote},
 		{RETROK_PERIOD, KBD_period}, {RETROK_COMMA, KBD_comma}, {RETROK_SLASH, KBD_slash},
-		{RETROK_PRINT, KBD_printscreen}, {RETROK_SYSREQ, KBD_printscreen}, {RETROK_PAUSE, KBD_pause},
+		{RETROK_PRINT, KBD_printscreen}, {RETROK_PRINT, KBD_printscreen}, {RETROK_PAUSE, KBD_pause},
 		{RETROK_INSERT, KBD_insert}, {RETROK_HOME, KBD_home}, {RETROK_PAGEUP, KBD_pageup},
 		{RETROK_PAGEDOWN, KBD_pagedown}, {RETROK_DELETE, KBD_delete}, {RETROK_END, KBD_end},
 		{RETROK_LEFT, KBD_left}, {RETROK_UP, KBD_up}, {RETROK_DOWN, KBD_down}, {RETROK_RIGHT, KBD_right},
-- 
2.37.0.windows.1

